primitives/messages/src/source_chain.rs

```diff
@@ -137,9 +149,14 @@ pub struct NoopMessagesBridge;
 
 impl<Payload> MessagesBridge<Payload> for NoopMessagesBridge {
```

Comment:
Not a part of this PR - sorry. But I think this `NoopMessagesBridge` isn't used anywhere anymore - so if it causes additional issues, just remove it.